tox.ini

```diff
@@ -74,6 +74,7 @@ deps =
 
 changedir = python/metatensor-core
 commands =
+    # install metatensor-operations
```

Comment:
We do not install `metatensor-operations` in these tests

Reply:
Oops sorry, I was messing around with some stuff and forgot to remove it

Reply:
I think I removed it